server/api.js

```diff
@@ -75,24 +75,28 @@ const api = () => {
     });
   };
 
-///////////////getNewMessage ////////////
+//----getNewMessage----//
 
-    const getNewMessage = async (req, res) => {
-    try {  
-      const userIdForMassage = parseInt(req.params.userId);
-      const result = await pool.query(`SELECT m.content, times_stamp FROM messages m WHERE m.user_id=$1`, 
-      [userIdForMassage]
-      );
+const getNewMessage = async (req, res) => {
+  try {  
+    const massagesByEventId = req.query.event;
+    const result = await pool.query(`SELECT * FROM messages m WHERE m.event_id=$1`, 
+    [massagesByEventId]
+    );
+    const resultArr = result.rows;
 
-      const mesggageArray = result.rows;
-      const lastMessage =  mesggageArray[mesggageArray.length - 1];
+    if(resultArr.length === 0){
```

Comment:
Let's not consider this case (`resultArr.length === 0`) and instead always return `resultArr`
```
res.send(resultArr)
```